neurodamus/utils/pyutils.py

```diff
@@ -209,3 +210,19 @@ def gen_ranges(limit, blocklen, low=0, block_increase_rate=1):
         yield low, high
         low = high
         blocklen = int(blocklen * block_increase_rate)
+
+
+def check_dir(d):
```

Comment:
Is this function used?

Reply:
it will. I need to switch the paths in node to properties.

Reply:
removed